.github/workflows/pull-request.yml

```diff
@@ -35,7 +35,7 @@ jobs:
 
       - name: Test Suite
         run: |
-          composer test
+          composer check
```

Comment:
Wait, what is ci.yml then? If this is for CI?

Reply:
I've traditionally split these up in my projects. But IDC how it gets done, as long as its done.

Reply:
I've traditionally split these up in my projects, but IDC how it gets done just that it gets done.

Reply:
Would be good of those would be the same as for the majority of plugins
One file, and one badge in readme

Reply:
Give that a look.